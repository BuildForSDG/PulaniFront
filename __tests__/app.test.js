import React from 'react';
import ReactDOM from 'react-dom';
import App from '../src/app';

describe('app module', () => {
	test('it exists', () => {
		expect(App).toBeDefined();
	});

	test('it renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<App />, div);
		ReactDOM.unmountComponentAtNode(div);
	});
});
