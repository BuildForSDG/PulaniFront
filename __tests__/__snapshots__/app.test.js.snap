// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app module that it renders without crashing 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  >
    Loading...
  </div>
</div>
`;
